src/main/java/uk/gov/digital/ho/hocs/audit/core/LogEvent.java

```diff
@@ -1,25 +1,42 @@
 package uk.gov.digital.ho.hocs.audit.core;
 
 public enum LogEvent {
-
     AUDIT_EVENT_DELETED,
     AUDIT_EVENT_CREATION_FAILED,
-    AUDIT_RECORD_NOT_FOUND,
     CSV_CUSTOM_CONVERTER_FAILURE,
     CSV_EXPORT_COMPLETE,
-    CSV_EXPORT_FAILURE,
     CSV_EXPORT_START,
-    CSV_RECORD_EXPORT_FAILURE,
-    INVALID_AUDIT_PAYLOAD_STORED,
     INVALID_CASE_TYPE_SPECIFIED,
     INVALID_PARAMETER_SPECIFIED,
     JSON_PARSE_EXCEPTION,
     MISSING_REQUEST_PARAMETER,
     NON_EXISTENT_VARIABLE,
     REFRESH_MATERIALISED_VIEW,
-    REST_CLIENT_EXCEPTION,
-    UNAUTHORISED_ACCESS,
-    UNCAUGHT_EXCEPTION;
+    UNCAUGHT_EXCEPTION,
+    USER_ROW_EXPORT_FAILURE,
```

Comment:
I wonder if reversing the order of these -- `EXPORT_FAILURE_USER_ROW` -- would make it easier to find in logs

Reply:
I've renamed the events, as for the first point, I'm reluctant to log user input (query string) but do we need to audit extracts maybe?